ui/app/components/job-status/panel/steady.js

```diff
@@ -25,10 +25,10 @@ export default class JobStatusPanelSteadyComponent extends Component {
     let availableSlotsToFill = this.totalAllocs;
     // Only fill up to 100% of totalAllocs. Once we've filled up, we can stop counting.
     let allocationsOfShowableType = this.allocTypes.reduce((blocks, type) => {
-      const jobAllocsOfType = this.args.job.allocations.filterBy(
-        'clientStatus',
-        type.label
-      );
+      const jobAllocsOfType = this.args.job.allocations
+        .sortBy('jobVersion') // Try counting from latest deployment's allocs and work backwards if needed
```

Comment:
Side-effect; will count backwards from the most recent by job version, rather than iterating through them in their default order.